src/core/Akka/Actor/Internal/ActorSystemImpl.cs

```diff
@@ -229,7 +229,7 @@ private void WarnIfJsonIsDefaultSerializer()
             {
                 Log.Warning($"NewtonSoftJsonSerializer has been detected as a default serializer. " +
                             $"It will be obsoleted in Akka.NET starting from version 1.5 in the favor of Hyperion " +
-                            $"(for more info visit: http://getakka.net/docs/Serialization#how-to-setup-wire-as-default-serializer ). " +
+                            $"(for more info visit: http://getakka.net/docs/Serialization#how-to-setup-hyperion-as-default-serializer ). " +
```

Comment:
This URL is going to change once https://github.com/akkadotnet/getakka.net/pull/223 is merged